### PR TITLE
[#17423] fix: update sync flow

### LINKED ISSUE #17423: Onboarding: updated sync flow
1) Replaced the copy underneath the scanning area to say: "Ensure both devices are on the same network"
Screenshot_2023-09-26_at_10 38 03
2) Replaced the theme switcher at the end of the onboarding flow with just a welcome screen with no interaction (no illustration so far) 

Screenshot_2023-09-26_at_10 38 20

Requested [here](https://discord.com/channels/624634427930312714/1146065551060389943/1156167230782967839)

### PR DESCRIPTION
**Pull request by mohsen-ghafouri**
fixes #17423

### Summary

-  Replaced the copy underneath the scanning area to say: "Ensure both devices are on the same network"
- Replaced the theme switcher at the end of the onboarding flow with just a welcome screen with no interaction ( i couldn't find any theme switcher in the welcome screen) 
- added welcome illustration 

### screenshots
<div>

</div>

status: ready

### CODE REVIEW

**Review by J-Son89 — APPROVED** [review]

**Review by ulisesmac — APPROVED** [review]

**Comment by mohsen-ghafouri** [comment]
@status-im/mobile-qa all tests failed with this error "Button by accessibility id: `welcome-button` is not found" but we still have this button and i didn't touch it, any idea why tests failed?

**Comment by churik** [comment]
@mohsen-ghafouri illustration is not centered on Android and button is not accessible on emulators, this is the reason for failed tests.
Physical Android (Google Pixel 7):
IMAGE 2023-10-25 11:59:04

Emulator:

image

**Comment by mohsen-ghafouri** [comment]
@J-Son89 could you please review it again?

**Comment by mohsen-ghafouri** [comment]
Android Screen shot

File: src/status_im2/contexts/onboarding/welcome/view.cljs [hunk 1]
```diff
@@ -40,10 +44,14 @@
...
        :on-press   #(rf/dispatch [:navigate-back-within-stack :enable-notifications])}]
      [page-title]
-     [rn/view {:style style/page-illustration}
-      [quo/text
-       "Illustration here"]]
+     [rn/image
+      {:style  (style/page-illustration (:width window))
+       :source (resources/get-image :welcome-illustration)}]
      [rn/view {:style (style/buttons insets)}
+      (when (< (:height window) 700)
```
Comment (J-Son89) [inline src/status_im2/contexts/onboarding/welcome/view.cljs]: we have a function `small-screen?` in `react-native.core`

**Review by J-Son89 — APPROVED** [review]

File: src/status_im2/contexts/onboarding/welcome/view.cljs (hunk 1 above)
Comment (mohsen-ghafouri) [reply src/status_im2/contexts/onboarding/welcome/view.cljs]: Cool, done

**Review by jo-mut — APPROVED** [review]

**Comment by mohsen-ghafouri** [comment]
@churik I resolved the issue and move it again to E2E test , but still no clue about result. could you please check

**Comment by mohsen-ghafouri** [comment]
@churik please check again

**Comment by churik** [comment]
@Francesca-G 

Iphone14: 
IMAGE 2023-10-26 12:46:04

Google Pixel 7:
IMAGE 2023-10-26 12:46:07


REady to merge from my POV

**Review by Francesca-G — APPROVED** [review]
Looks good ✨

**Pull request merged**